actors/miner/src/deadline_info.rs

```diff
@@ -85,6 +86,24 @@ impl DeadlineInfo {
         }
     }
 
+    pub fn new_from_policy(
```

Comment:
This appears to be a duplicate of `new_deadline_info` from actors/miner/src/deadline.rs.